resources/build/build-utils.sh

```diff
@@ -74,7 +74,7 @@ function findVersion() {
         VERSION_TAG=
     fi
 
-    if [ -z "${TEAMCITY_VERSION-}" -a -z "${JENKINS_HOME-}" ]; then
+    if [ -z "${TEAMCITY_VERSION-}" ]; then
```

Comment:
Do we need a GHA condition here?

Reply:
→ https://github.com/keymanapp/keyman/pull/9400